### ISSUE TITLE
Feature request: Add feature to use and switching between multiple accounts

### ISSUE THREAD

**Issue by jahway603** [etesync/etesync-notes#188]
* Implement configuration of multiple account usage
* Implement switching between these multiple accounts in the UI

**Comment by tasn** [comment]
Yeah, it's useful, and probably not super hard to do. A lot of the code assumes a single account but was designed with multi-account in mind.
